src/esp/assets/ResourceManager.cpp

```diff
@@ -286,21 +286,46 @@ PhysicsManagerAttributes ResourceManager::loadPhysicsConfig(
   std::string configDirectory =
       physicsFilename.substr(0, physicsFilename.find_last_of("/"));
   // load the rigid object library metadata (no physics init yet...)
-  // NOTE: expect relative paths to the global config
   if (scenePhysicsConfig.HasMember("rigid object paths")) {
     if (scenePhysicsConfig["rigid object paths"].IsArray()) {
       physicsManagerAttributes.setVecStrings("objectLibraryPaths",
                                              std::vector<std::string>());
       for (rapidjson::SizeType i = 0;
            i < scenePhysicsConfig["rigid object paths"].Size(); i++) {
         if (scenePhysicsConfig["rigid object paths"][i].IsString()) {
-          // 1: read the filename (relative path)
-          std::string objPhysPropertiesFilename =
-              configDirectory + "/" +
-              scenePhysicsConfig["rigid object paths"][i].GetString() +
-              ".phys_properties.json";
-          physicsManagerAttributes.appendVecStrings("objectLibraryPaths",
-                                                    objPhysPropertiesFilename);
+          std::string filename =
+              scenePhysicsConfig["rigid object paths"][i].GetString();
+          std::string directory = configDirectory + "/";
+          // if it ends with "/", treat as directory and try to load the whole
+          // thing
+          if (filename.back() == '/') {
```

Comment:
If you want a more reliable check, there's a new `Directory::isDirectory()` that got added in mosra/corrade@de69dbc62a8e62d0494c50a32633b8d0e806d667 last week. You may need to pull the corrade submodule for that, tho :)

Reply:
Yeah.  I wold definitely recommend pulling in that update.  Forcing people to add a trailing `/` for directory mode seems less than ideal.